### PR TITLE
[NativeAOT-LLVM] Move dotnet.js to aotsdk

### PR DESCRIPTION
**Pull request by maraf**
Contributes to https://github.com/dotnet/runtimelab/issues/2434

### CODE REVIEW

**Comment by maraf** [comment]
cc @dotnet/nativeaot-llvm

File: eng/liveBuilds.targets
Comment (SingleAccretion) [inline eng/liveBuilds.targets]: This should not be needed anymore. Also, `Microsoft.DotNet.ILCompiler.LLVM.pkgproj` needs to be fixed to not copy these files into `framework/`.

File: src/coreclr/nativeaot/BuildIntegration/DotNetJsApi.targets
Comment (SingleAccretion) [inline src/coreclr/nativeaot/BuildIntegration/DotNetJsApi.targets]: I suspect this needs tweaks to `Microsoft.DotNet.ILCompiler.LLVM.pkgproj` to preserve the `src/es6`. Alternatively, it needs fixes here to work with the flat hierarchy (which would be simpler if it's possible).

**Review by SingleAccretion — COMMENTED** [review]
Please also verify manually that the package still works and has the expected layout before merging.

File: src/coreclr/nativeaot/BuildIntegration/DotNetJsApi.targets
Comment (maraf) [reply src/coreclr/nativeaot/BuildIntegration/DotNetJsApi.targets]: Good catch! I forgot changes to pkgproj in stash

**Comment by maraf** [comment]
image

**Review by SingleAccretion — APPROVED** [review]
Changes LGTM assuming the package works. Thank you!

**Pull request merged**
